src/App.tsx

```diff
@@ -1,6 +1,11 @@
 import { Routes, Route } from "react-router-dom";
+
+import Banner from "./pages/Banner";
 import Login from "./pages/Login";
 import SignIn from "./pages/Sign";
+import StudentsPage from "./pages/Students";
+import TeachersPage from "./pages/Teachers";
```

Comment:
We can create an index.ts file and import all pages there and export from the index file. And use pages here something like

```
import { Banner, Login, SignIn } from "./pages";
```